scripts/archive/build-release-archives.sh

```diff
@@ -105,8 +107,12 @@ fi
 mkdir docker_build 
 mv coda-*.deb docker_build/.
 
-echo "$DOCKER_PASSWORD" | docker login --username $DOCKER_USERNAME --password-stdin
+if [ -z $DOCKER_PASSWORD ]; then
```

Comment:
This should probably have a`set +u` too

Reply:
We don't need this check at all actually because we'll have exited above first -- fixed that